src/idiomatic/foundations-api-design/predictable-api/naming-conventions/11-unchecked.md

```diff
@@ -0,0 +1,25 @@
+---
+minutes: 2
+---
+
+# Unchecked
+
+Component for methods and functions where users need to manage safety manually.
+
+```rust
+NonNull::new_unchecked
+Vec::split_at_unchecked
```

Comment:
Please show the signatures for both functions in the pair:

`NonNull::new()` / `NonNull::new_unchecked()`

`Vec::split_at()` / `Vec::split_at_unchecked()`